.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,92 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[State]: Title of the bug"
+labels: ''
```

Comment:
```suggestion
labels: 'state | template'
```
shouldn't we mention to use either state or template label? Wherever it fits best?